web/src/components/MemoEditor.tsx

```diff
@@ -404,6 +437,14 @@ const MemoEditor: React.FC = () => {
           <button className="action-btn" onClick={handleFullscreenBtnClick}>
             {state.fullscreen ? <Icon.Minimize className="icon-img" /> : <Icon.Maximize className="icon-img" />}
           </button>
+          <label className="form-label selector">
```

Comment:
I think it will cause overflow problem. Please take a look into the mobile style.

Reply:
Yes, I ignored it. Any good suggestions? I don't know how to handle it.

For mobile style, how about placing the selector below the buttons?

Reply:
How about put the selector in the following place?
<img width="653" alt="image" src="https://user-images.githubusercontent.com/24653555/200158028-2ab6a0f3-67b6-4ff0-bf23-28f199b568ac.png">

Reply:
I tried two, but neither seemed perfect.

1.

creating:
![image](https://user-images.githubusercontent.com/31177490/200159078-d9352557-7488-45db-a1ca-168526d09628.png)

The selector overwrited the text.

updating:
![image](https://user-images.githubusercontent.com/31177490/200159045-5ab093be-4ffe-4e85-8ae6-25290704503d.png)


2.

creating:
![image](https://user-images.githubusercontent.com/31177490/200159216-60786f3d-69fe-43b0-b449-800c32139693.png)

A large space.

updating:
![image](https://user-images.githubusercontent.com/31177490/200159239-99f59f9a-eee6-4a24-8a51-595cee91534f.png)

Reply:
I think we shouldn't show this when creating new memo as we already has a default visibility option. 

So let's show the visibility selector only when editing memo.

Reply:
> I tried two, but neither seemed perfect.

![image](https://user-images.githubusercontent.com/20352705/200167080-218da1d9-1c4f-496b-ba3b-a8ee20f6abba.png)
Not here?